docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -238,6 +240,106 @@ per output type, which enables loadbalancing.
 |Supported
 |===
 
+[discrete]
+[[additional-capabilties-beats-and-agent]]
+== Additional capabilities
+
+The following table shows the additional capabilities supported by {beats} and the {agent} in {version}:
+
+
+[options,header]
+|===
+|Item |{beats} |{fleet}-managed {agent} |Standalone {agent}
+
+|Single agent for all use cases
+|{n}
+|{y}
+|{y}
+
+|Install integrations from Kibana UI or API
+|{n}
+|{y}
+|{y}
+
+|Configure from Integrations UI
+|{n}
+|{y}
+|{y} (optional)
+
+|Central, remote agent policy management
+|{n}
+|{y}
+|{n}
+
+|Central, remote agent binary upgrades
+|{n}
+|{y}
+|{n}
+
+|Choose which module or integration assets to add
+|{n}
+|{y}
+|{y}
+
+|Auto-generated {es} API keys
+|{n}
+|{y}
+|{n}
+
+|Auto-generate minimal {es} privileges on the edge
+|{n}
+|{y}
+|{y} (when using Integrations UI)
+
+|Data streams support
+|{n}
+|{y}
+|{y}
+
+|Variables and input conditions
+|{n}
+|{y}
+|{y}
+
+|Requires superuser role to add integrations and agent polices
+|{n}
+|{y} 
+|{n} (it's optional to use the Integrations UI)
+
+|Air-gapped network support
+|{y}
+|{n}
+|{y}
+
+|Run without root on host
+|{y}
+|{n}
+|{y}
+
+|Mulitple outputs
+|{y}
+|{n}
+|{y}
+
+|Separate monitoring cluster 
+|{y}
+|{n}
+|{y}
+
+|Keystore for secrets
+|{y}
+|{n}
+|{n}
+
+|Cloned machine images or virtual desktops
+|{y}
+|{n}
+|{n}
+
+|===
+
+Also, both {beats} and {agent} standalone work well with version control, IaaC, and change management processes because the configuration is a simple text file. {fleet} only supports these processes through its REST API.
```

Comment:
Can we link to fleet API docs?

Reply:
We have an open issue to publish API docs here https://github.com/elastic/obs-dc-team/issues/308

Reply:
I suggest adding a link after we publish the docs